Makefile

```diff
@@ -1,26 +1,32 @@
-.PHONY: all clear
+.PHONY: tempdir all clear
+
+TEX = pdflatex -interaction=nonstopmode -shell-escape
+BIB = biber
 
-doc_name = tese_lncc
-TEX = pdflatex -interaction=nonstopmode
-BIB = "bibtex"
 pwd := $(shell pwd)
 build_dir := $(shell mktemp -d)
 
-all:	clear
-	@echo "Building at" $(build_dir)
-	# mirror the src dir at temp build dir
-	( find $(pwd) -name "tex/*" -type f -exec ln -s {} $(build_dir) \;)
-	( find $(pwd) -name "fig/*" -type f -exec ln -s {} $(build_dir) \;)
-	( find $(pwd) -name "*" -type f -exec ln -s {} $(build_dir) \;)
-	# build at temp build dir
-	-( cd $(build_dir) && $(TEX) -draftmode $(doc_name).tex )
-	-( cd $(build_dir) && $(BIB) $(doc_name).aux )	
-	-( cd $(build_dir) && $(TEX) -draftmode $(doc_name).tex )
-	-( cd $(build_dir) && $(TEX) $(doc_name).tex )
-	# copy the output to here
-	-( cd $(build_dir) && cp -f $(doc_name).pdf $(pwd) )
+all: tese_lncc.pdf
+
+.ONESHELL:
+tempdir: $(build_dir)	
```

Comment:
Aqui, para remediar o erro, antes de linkar simbolicamente, sugiro fazer: `tempdir: clear $(build_dir)`

Reply:
pode deixar o `clear`, mas o diretório `build_dir` é criado no `\tmp` a cada chamada do `make`, é necessário limpá-lo?

Reply:
a dependencia `clear` deve ir no `all: clear tese_lncc.pdf slides.pdf poster.pdf`

Reply:
Incluir a dependência `clear` em `all` gera status de erro.

Reply:
Um PDF caro de se produzir, por exemplo, não deve ser apagado sem o consentimento do usuário.  Forçar a remoção da versão anterior do PDF como requisito para produzir a próxima versão não me parece boa coisa. Acedito que o `clear` deva ser acionado manualmente pelo usuário, consciente de sua ação.

Reply:
> Um PDF caro de se produzir, por exemplo, não deve ser apagado sem o consentimento do usuário. Forçar a remoção da versão anterior do PDF como requisito para produzir a próxima versão não me parece boa coisa. Acedito que o `clear` deva ser acionado manualmente pelo usuário, consciente de sua ação.

De fato, faz sentido. Mas pq se rodaria o Make se não para construir o pdf? Há outras opções ali?